### PR TITLE
FAT-5014

### PR DESCRIPTION
**Pull request by re-jas**
- https://issues.folio.org/browse/FAT-5014
- [Allure patron block tests](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1803/allure/)
- [Allure C9217](https://jenkins-aws.indexdata.com/job/Testing/job/Cypress_tests/1931/allure/)

### CODE REVIEW

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js [hunk 1]
```diff
@@ -215,36 +220,52 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
...
-            id: uuid(),
-            itemBarcode: item.barcode,
-            loanDate: moment.utc().format(),
-            servicePointId: testData.userServicePoint.id,
-            userBarcode: userData.barcode,
-          });
-        });
+        cy.login(userData.username, userData.password);
+        cy.visit(SettingsMenu.conditionsPath);
+        Conditions.waitLoading();
```
Comment (penfriend) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js:225]: please check if it is possible here to substitute three lines in one line

Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-removed-after-lost-item-returned-renewed.cy.js:225]: unfortunately it won't work. When you go directly to this section, it is not displayed properly

**Review by MaksimTerekhov — APPROVED** [review]

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js [hunk 2]
```diff
@@ -274,30 +302,67 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
...
+      UsersCard.waitLoading();
+      Users.checkIsPatronBlocked(blockMessage, 'Borrowing, Renewals, Requests');
+
+      const itemForRenew = itemsData.itemsWithSeparateInstance[Math.floor(Math.random() * 5)];
+      UsersCard.openLoans();
+      UsersCard.showOpenedLoans();
+      UserLoans.openLoan(itemForRenew.barcode);
+      UserLoans.renewItem(itemForRenew.barcode, true);
+      Renewals.renewBlockedPatron(renewComment);
+      RenewConfirmationModal.confirmRenewOverrideItem();
```
Comment (penfriend) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js:330]: Let's check here that this modal exists before the button click

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js [hunk 3]
```diff
@@ -274,30 +302,67 @@ describe('Patron Block: Maximum outstanding fee/fine balance', () => {
...
+      OverrideAndRenewModal.confirmOverrideItem();
```
Comment (penfriend) [inline cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js:331]: Let's check here that this modal exists before the button click

**Review by penfriend — APPROVED** [review]

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js (hunk 3 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js:332]: done

File: cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js (hunk 2 above)
Comment (re-jas) [reply cypress/e2e/settings/users/patron-block/patron-block-maximum-fee-fine-balance-and-lost-items-removed-after-lost-item-returned-renewed.cy.js:331]: done

**Pull request merged**